locale/de_DE/LC_MESSAGES/OpenInExplorer.po

```diff
@@ -0,0 +1,38 @@
+# OpenInExplorer Darktable Plug-In.
+# Copyright (C) 2020 Volker Lenhardt
+# This file is distributed under the same license as the PACKAGE package.
+# Volker Lenhardt <volker.lenhardt@unitybox.de>, 2020.
+#
+#, fuzzy
+msgid ""
+msgstr ""
+"Project-Id-Version: OpenInExplorer\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2020-04-14 14:35+0200\n"
+"PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
+"Last-Translator: Volker Lenhardt <volker.lenhardt@unitybox.de>\n"
+"Language-Team: LANGUAGE <LL@li.org>\n"
+"Language: de_DE\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+
+#: OpenInExplorer.lua:59
+msgid "OpenInExplorer plug-in only supports Windows and Linux at this time"
+msgstr "OpenInExplorer-Plug-in kann nur unter macOS, Linux oder Windows laufen"
+
+#: OpenInExplorer.lua:75
+msgid "Please select an image"
+msgstr "Bitte mindestens 1 Bild auswählen"
```

Comment:
Ich würde die "1" durch das Wort "ein" ersetzen.

Reply:
"1" wird beim Lesen als Numeral immer betont wahrgenommen (glaube ich), wohingegen "ein" vielleicht eher als unbestimmter Artikel die Betonung auf "Bild" setzt, obwohl hier auch keine Alternative zu einem Bild möglich ist. "Bitte ein Bild auswählen" könnte als Beschränkung der Auswahl verstanden werden. "Bitte ein Bild oder mehrere Bilder auswählen" gefällt mir schon gar nicht.

Reply:
Und was hältst du von: "Es wurde kein Bild ausgewählt"?

Reply:
OK, akzeptiert. Denn wenn man es ganz genau nimmt, wäre "1 Bild" ja "eins Bild". In einem durchgehenden Text gewiss ein Stilfehler.